### DISCUSSION TITLE
PROBLEM : How to set Dark Theme as default?

### DISCUSSION THREAD

**Discussion by MGeurts** [tasteui/tasteui#530] [Q&A]
How do I set the Dark Theme as default?

The documentation states this :
```
<html ...
      x-data="tallstackui_darkTheme('other-name', true)" 
      ...>
    <!-- ... -->
</html>
```

But I don't want to rename the theme. I only want to set dark mode as default?
Is the true option addressable as a named option?
Please clarify the documentation.

Thanks.

**Comment by devajmeireles** [comment]
```blade
<html ...
      x-data="tallstackui_darkTheme(null, true)" 
      ...>
    <!-- ... -->
</html>
```

**Comment by MGeurts** [comment]
There's a problem with the solution suggested above. It sets the dark theme as default, like promised.
**But after switching the theme to light with the "Theme Switch", this is not remembered by the browser.**
This means navigating to another page sets the theme to dark again.

Please help. Thanks.
